apollo-adapters/api/apollo-adapters.api

```diff
@@ -18,6 +18,22 @@ public final class com/apollographql/apollo3/adapter/DateAdapter : com/apollogra
 	public fun toJson (Lcom/apollographql/apollo3/api/json/JsonWriter;Lcom/apollographql/apollo3/api/CustomScalarAdapters;Ljava/util/Date;)V
 }
 
+public final class com/apollographql/apollo3/adapter/GuavaOptionalAdapter : com/apollographql/apollo3/api/Adapter {
+	public fun <init> (Lcom/apollographql/apollo3/api/Adapter;)V
+	public fun fromJson (Lcom/apollographql/apollo3/api/json/JsonReader;Lcom/apollographql/apollo3/api/CustomScalarAdapters;)Lcom/google/common/base/Optional;
+	public synthetic fun fromJson (Lcom/apollographql/apollo3/api/json/JsonReader;Lcom/apollographql/apollo3/api/CustomScalarAdapters;)Ljava/lang/Object;
+	public fun toJson (Lcom/apollographql/apollo3/api/json/JsonWriter;Lcom/apollographql/apollo3/api/CustomScalarAdapters;Lcom/google/common/base/Optional;)V
+	public synthetic fun toJson (Lcom/apollographql/apollo3/api/json/JsonWriter;Lcom/apollographql/apollo3/api/CustomScalarAdapters;Ljava/lang/Object;)V
+}
```

Comment:
When we have `apollo-runtime-java`, we should put it there instead of `apollo-adapters`. Maybe now is even a good time to create the module?

Reply:
Oh gosh, nevermind, it should be more like `apollo-api-java` I guess 🤔